modules/meilisearch/meilisearch.go

```diff
@@ -0,0 +1,131 @@
+package meilisearch
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/docker/docker/api/types/container"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+const (
+	defaultMasterKey = "just-a-master-key-for-test"
+	defaultHTTPPort  = "7700/tcp"
+)
+
+// MeilisearchContainer represents the Meilisearch container type used in the module
+type MeilisearchContainer struct {
+	testcontainers.Container
+	MasterKey string
+}
+
+// Deprecated: use Run instead
+// RunContainer creates an instance of the Meilisearch container type
+func RunContainer(ctx context.Context, opts ...testcontainers.ContainerCustomizer) (*MeilisearchContainer, error) {
+	return Run(ctx, "getmeili/meilisearch:v1.10.3", opts...)
+}
```

Comment:
Seeing this block of code, I guess you copy-pasted some other module. Could you run the code generation tool for creating modules, as explained [here](https://golang.testcontainers.org/modules/#testcontainers-for-go-modules)?

It informs of the different alternatives to host the module (here in the core or in your own GH user as a community module), also creating the docs pages for the new module. Finally, it will update the files to include the new module in the CI, among others.

I'd suggest you run the tool first, checking that your code does not include `Deprecated` code (the tool does not include it)

Thanks!

Reply:
@mdelapenya yes true I copied paste from OpenSearch 😄

Ok sure I will run and re-request your review. Thank you @mdelapenya for the comment.

Reply:
I have a commit with the scaffolding, but I cannot push it because this branch is your main branch. I'll try to submit a PR to your branch so you can incorporate it into this one

Reply:
Sure @mdelapenya